src/main/java/training/subsystems/Elbow/ElbowFactory.java

```diff
@@ -0,0 +1,22 @@
+package training.subsystems.Elbow;
+
+import training.subsystems.Elbow.NeoElbow.NeoElbow;
+import training.subsystems.Elbow.SimulationElbow.SimulationElbow;
+
+public class ElbowFactory {
+    public static IElbow create() {
+        IElbow elbow;
+        switch (ElbowConstants.ROBOT_TYPE) {
```

Comment:
```suggestion
        switch (Robot.ROBOT_TYPE) {
```
this needs to be in robot